src/aind_registration_evaluation/util/excelOutput.py

```diff
@@ -0,0 +1,49 @@
+import os
+import numpy as np
+import pandas as pd
+
+def create_excel_file(file_path):
```

Comment:
There utilities could be added to the utils package. Please make sure for new files you follow the **file_name.py** convention.